src/task-manager.ts

```diff
@@ -30,15 +23,14 @@ export class TaskManager implements ItemFlagTaskManager {
     return GetFlagsTask.name;
   }
 
-  createCreateTask(member: Member, data: Partial<ItemFlag>, itemId: string): CreateItemFlagTask {
-    return new CreateItemFlagTask(
-      member,
-      data,
-      itemId,
-      this.itemService,
-      this.itemMembershipService,
-      this.itemFlagService,
-    );
+  createCreateTaskSequence(
+    member: Member,
+    data: Partial<ItemFlag>,
+    itemId: string,
+  ): Task<Actor, unknown>[] {
+    const tasks = this.itemTaskManager.createGetTaskSequence(member, itemId);
```

Comment:
This is just to check the access right? That's why `t3.getInput` is not needed

Reply:
It's just to check the access indeed, and t3 doesn't need any data from the access!